nix/m1-support/mesa/default.nix

```diff
@@ -0,0 +1,13 @@
+{ config, ... }:
+{
+  config = {
```

Comment:
This wasn't going to work as a module until it got this top-level `config` attrset! Once I added that, these settings were applied to my system configuration.